__tests__/retry-utils.test.ts

```diff
@@ -0,0 +1,166 @@
+import { describe, it, expect, afterEach } from 'vitest'
+import {
+  withRetryFetch,
+  withRetryAnthropic,
+  withRetry
+} from '../src/utils/retry.ts'
+
+function makeRes(
```

Comment:
Je serais pas contre un renommage un poil plus clair :

```suggestion
function makeResponse(
```